### DISCUSSION TITLE
can i limit the threads of parallel?

### DISCUSSION THREAD

**Discussion by zzzgit** [gulpjs/gulp#2670] [Help] [answered]
Say I have a parallel schedule like this: 
`let schedule = parallel(task1, task2,task2...task999)`

if the 999 tasks run simultaneously, the CPU usage rate will get high. So, is there a way to limit the concurrence number? That is to say, like, provide 3 threads for 3 tasks respectively, after one task is achieved, the corresponding thread will be released, then assign another new task to the free thread.

**Comment by phated** [comment, marked as answer]
Not yet! There was an attempt to do this at https://github.com/gulpjs/now-and-later/pull/15 but the API changes didn't get added. Hoping to include that in v5
